phpseclib/File/X509.php

```diff
@@ -429,12 +430,8 @@ public function __construct()
      * Load X.509 certificate
      *
      * Returns an associative array describing the X.509 cert or a false if the cert failed to load
-     *
-     * @param string $cert
-     * @param int $mode
-     * @return mixed
      */
-    public function loadX509($cert, $mode = self::FORMAT_AUTO_DETECT)
+    public function loadX509(string $cert, int $mode = self::FORMAT_AUTO_DETECT)
```

Comment:
This needs to be able to accept arrays. The code right below it does an `is_array($cert)` check and then does some specific things if it is an array. https://github.com/phpseclib/phpseclib/commit/3f9aa1ad6a6a1ca4ce98fb1f712f4c8e0a471bf6 is the change that added that.

I suppose we could drop support for arrays but idk that I see an overwhelming need to